raiden/connection_manager.py

```diff
@@ -144,9 +167,9 @@ def close_all(self, only_receiving=True):
             channels_to_close = (
                 self.receiving_channels[:] if only_receiving else self.open_channels[:]
             )
-            for channel in channels_to_close:
+            for channel_ in channels_to_close:
```

Comment:
What's the underscore here for?

Reply:
name collision

Reply:
Yep but with which channel variable? From an initial look at it I could not figure it out.

Reply:
in a previous version the channel module was imported